go/ngql/types.go

```diff
@@ -730,6 +848,52 @@ func MaybeGetScalar(v types.Value) interface{} {
 	return v
 }
 
+func argToNomsValue(arg interface{}, nomsType *types.Type) types.Value {
+	switch nomsType.Kind() {
+	case types.BoolKind:
+		return types.Bool(arg.(bool))
+	case types.NumberKind:
+		if i, ok := arg.(int); ok {
+			return types.Number(float64(i))
```

Comment:
I'm not sure you need the extra `float64` cast, since number types can be cast to each other, and types.Number is just a number type.